src/bot/commands/rank.ts

```diff
@@ -264,7 +264,7 @@ async function getTopChannels(
   );
 
   if (!guildMemberTopChannels || guildMemberTopChannels.length == 0)
-    return ${t('rank.noEntries')};
+    return `${t('rank.noEntries')}`;
```

Comment:
This can be reduced to just
```suggestion
    return t('rank.noEntries');
```